handler_test.go

```diff
@@ -1446,3 +1446,31 @@ func EncodeInputDef(name string, body []byte) (*internal.InputDefinition, error)
 	def.Name = name
 	return def, nil
 }
+
+func TestHandler_GetTimeStamp(t *testing.T) {
+	data := make(map[string]interface{})
+	timeField := "time"
+	data["time"] = "2017-03-20T19:35"
+	val, err := pilosa.GetTimeStamp(data, timeField)
+	if val != 1490038500 {
+		t.Fatalf("Timestamp is not set correctly for %s", data["time"])
+	}
+
+	data["int"] = 1490000000
+	val, err = pilosa.GetTimeStamp(data, "int")
+
```

Comment:
this doesn't matter but probably using `data["time"]` and `timeField` for the consistent?